Tasks/PackerBuildV0/Tests/L0.ts

```diff
@@ -504,6 +514,18 @@ describe('PackerBuild Suite', function() {
             done();
         });
 
+        it('Runs successfully for linux template with managed image', (done:MochaDone) => {
+            let tp = path.join(__dirname, 'L0LinuxManaged.js');
+            let tr : ttm.MockTestRunner = new ttm.MockTestRunner(tp);
+            tr.run();
+
+            assert(tr.invokedToolCount == 4, 'should have invoked tool four times. actual: ' + tr.invokedToolCount);
+            assert(tr.stderr.length == 0 || tr.errorIssues.length, 'should not have written to stderr');
+            assert(tr.succeeded, 'task should have succeeded');
+
```

Comment:
Add one more validation to ensure that image print in printed in output log

Reply:
added validation